### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by altendky**

### CODE REVIEW

**Comment by altendky** [comment]
Bonus, fixed a long line (good thing we weren't actually linting before...) and added testing on macOS.

**Comment by altendky** [comment]
@cdunklau

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+  pull_request:
```
Comment (cdunklau) [inline .github/workflows/ci.yml]: Is there a way to make this bit a bit clearer? At first I thought this meant you're *ignoring* PRs, but apparently it means all PRs.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      - name: Set up ${{ matrix.python.name }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python.action }}
+          architecture: x64
+      - name: Install Linux dependencies
+        if: matrix.system.platform == 'linux'
+        run: |
+          sudo apt-get update
+          sudo apt-get install xvfb
```
Comment (cdunklau) [inline .github/workflows/ci.yml]: I suspect some config somewhere makes apt-get assume yes implicitly, but maybe make it explicit: `sudo apt-get install -y xvfb`

Comment (altendky) [reply .github/workflows/ci.yml]: Yes, thanks.

Comment (cdunklau) [reply .github/workflows/ci.yml]: Yes.

**Pull request merged**